lua/wire/client/text_editor/modes/e2.lua

```diff
@@ -333,6 +333,9 @@ function EDITOR:SyntaxColorLine(row)
     local spaces = self:SkipPattern( " *" )
     if spaces then addToken( "comment", spaces ) end
 
+    local invalidInput = self:SkipPattern( "^[^A-Z:]" )
```

Comment:
This'll break packed arguments, i.e. `[AAA BBB]`. I think something like this will work `[^A-Za-z:%[%]]`.

Reply:
with this `^[^A-Z:%[%]]` something like this `function name(Table["string", number])` would still cause a crash

Reply:
`^[^A-Z:%[]` seems to work